cfgov/v1/atomic_elements/atoms.py

```diff
@@ -154,3 +155,33 @@ def clean(self, data):
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
         else:
             return data
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False)
```

Comment:
On a more important note, the value of the block is at risk of being changed inadvertently because the clean method will be called on each page save and the `get_unique_id()` will provide a different value since it's randomized at execution.